src/diffusers/pipelines/auto_pipeline.py

```diff
@@ -164,14 +163,6 @@ def get_model(pipeline_class_name):
         raise ValueError(f"AutoPipeline can't find a pipeline linked to {pipeline_class_name} for {model_name}")
 
 
-def _get_signature_keys(obj):
```

Comment:
I don't quiet remember why I decided to create a private function here for auto_pipeline instead of using the class method of `DiffusionPipeline`
https://github.com/huggingface/diffusers/blob/adcbe674a49af31415a1131174143d58c0e68d7d/src/diffusers/pipelines/pipeline_utils.py#L1927

but essentially they are the same, and when we introduced the optional components for the first time with IP-adapter PR, we updated the `_get_signature_keys` method on pipeline_utils and did not update this function so it is out of date. I remove this function here and use the one from the pipeline_utils instead